datadog/setupdatadog.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+#Set the enviroment for TORQHOME
+cd ..
```

Comment:
Do not change directory, use environment variables to find full paths.

Reply:
If this is run before the user has started their TorQ process and the environment need set for the first time, is there a nicer way to source setenv.sh correctly so that TORQHOME will be the correct path?

Reply:
It shouldn't be, running `setenv.sh` is explicitly the first instruction before doing anything with TorQ and it should be the same here.